src/payment/dto/create-payment.dto.ts

```diff
@@ -1 +1,7 @@
-export class CreatePaymentDto {}
+import { User } from 'src/user/entities/user.entity';
+
+export class CreatePaymentDto {
+  userId: string;
+  paymentMethod: string;
```

Comment:
proponuję zmienić w kodzie i w UMLu typ na jakiś enum albo arraya z polami:  karnet, gotówka, karta, mutlisport

Reply:
tak jak jest to zrobione w role.enum.ts z developa proponuję, czyli osobny plik i wtedy do niego: 

export enum PaymentMethodType {
  Pass = 'pass',
  Cash = 'cash',
  Card= 'card',
  SportCard= 'sportCard',
}

Reply:
dobry pomysł to jest, też o tym pomyślałam, ale tak jak Darek mówił nie wiem czy nie zająć się tym później, to już Adam up to you teraz ;) ale możesz już później (po wtorku xd), i co do tego enum to mam zastrzeżenie małe, bo wpisałabym `SportCard = "sport card"`, bo przy możliwości wpisywania tych metod z palca, raczej każdy pisałby to ze spacją. Pewnie będzie to jako wybór z rozwijalnej listy albo coś takiego, ale na razie tego jeszcze nie mamy

Reply:
oki, już i tak po spotkaniu, więc teraz przed mergem już dobrze by było, żeby był enum ;)